src/components/Accordion/index.js

```diff
@@ -0,0 +1,47 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+
+const Accordion = ({ defaultExpandedPanel, children, ...passedProps }) => {
+  const [expandedPanel, setExpandedPanel] = useState(defaultExpandedPanel);
+
+  const handleToggle = (panel, onToggle) => (event, expanded) => {
+    const expandedPanel = expanded ? panel : false;
+    setExpandedPanel(expandedPanel);
```

Comment:
I don't think you should switch type of `expandedPanel` from a number to `false`. Maybe a number or `null` would be cleaner?

Reply:
You're right! I didn't realize that